tests/model_selection/dask_searchcv/test_model_selection.py

```diff
@@ -891,3 +898,56 @@ def test_mock_with_fit_param_raises():
 
     with pytest.raises(ValueError):
         clf.fit(X, y)
+
+
+@pytest.mark.parametrize(
```

Comment:
This test kind of subsumes the test in the sklearn PR because it tests the situation with one fold metric calculation and multiple fold metric calculations that need to be combined.  It does this by testing at the `dcv.GridSearchCV` level rather than at the level of functions in `dask_ml/model_selection/methods.py`.